### ISSUE TITLE
MTL website headshot 

### ISSUE THREAD

**Issue by ritahitching** [lzim/teampsd#793]
**Upload Casey Grysen Headshot:** 

@jamesmrollins please add Casey Grysen's headshot to be mtl.how website 

https://github.com/lzim/teampsd/tree/master/resources/headshots

**Comment by jamesmrollins** [comment]
Ok - I should be able to get this done Monday

**Comment by jamesmrollins** [comment]
@ritahitching I adjusted and uploaded this photo.  The aspect ratio and lighting were very difficult to work with.  The resulting picture in the circle is very blurry.  May I suggest we retake the photo?  I recommend a landscape orientation, that gives me the greatest flexibility.  Also, try taking the photo outside to make the best use of natural light (be sure the sun is behind you).  The indoor florescent lighting is just not very flattering. Finally, choose at least a medium resolution.  I can always down-sample the picture.  Thanks, James

**Comment by ritahitching** [comment]
@jamesmrollins thank you. I'll reach out to Casey and see if he has a better photo, and then circle back.

**Comment by lzim** [comment]
> @jamesmrollins thank you. I'll reach out to Casey and see if he has a better photo, and then circle back.
> 
> FYI @lzim @staceypark

@ritahitching I gave him the info at our 9AM meeting this morning. He said he would follow-up with you.

- I told him I’d love to have a new image up before our workshop at ISTSS next week.

**Comment by ritahitching** [comment]
@jamesmrollins @lzim 
I uploaded a new headshot of Casey to  github and created a pull request.  Once it's merged to the master you'll be able to find the photo at:
https://github.com/lzim/teampsd/tree/master/resources/headshots
thanks, Rita

**Comment by lzim** [comment]
> @jamesmrollins @lzim
> I uploaded a new headshot of Casey to github and created a pull request. Once it's merged to the master you'll be able to find the photo at:
> https://github.com/lzim/teampsd/tree/master/resources/headshots
> thanks, Rita

Thanks @ritahitching ! @jamesmrollins I merged the pull request.

@ritahitching and @jamesmrollins Do we have a headshot from Andrea Nevedal?
I don't see one at [mtl.how/team](https://mtl.how/team)

**Comment by ritahitching** [comment]
> > @jamesmrollins @lzim
> > I uploaded a new headshot of Casey to github and created a pull request. Once it's merged to the master you'll be able to find the photo at:
> > https://github.com/lzim/teampsd/tree/master/resources/headshots
> > thanks, Rita
> 
> Thanks @ritahitching ! @jamesmrollins I merged the pull request.
> 
> @ritahitching and @jamesmrollins Do we have a headshot from Andrea Nevedal?
> I don't see one at [mtl.how/team](https://mtl.how/team)

I'll ping Andrea again for a headshot and send an update.

**Comment by ritahitching** [comment]
@lzim @jamesmrollins photo of Andrea on GitHub, pending pull request merge. You'll find it in the same location as https://github.com/lzim/teampsd/tree/master/resources/headshots thanks

**Comment by jamesmrollins** [comment]
@lzim @ritahitching I updated the photos and also the slider maps.

**Comment by ritahitching** [comment]
@jamesmrollins thanks for updating the photos, will you please move Andrea's photo on the map to around the Detroit Michigan area, she's not based in Menlo Park.

**Comment by jamesmrollins** [comment]
@ritahitching sorry, it wasn't necessarily clear in her bio where she was located.  BTW, her photo was also a bit dicey.  If there is a better one, I could also update at the same time?

**Comment by ritahitching** [comment]
@jamesmrollins you are totally right, my apologies. I too thought she was in Menlo Park, it was Stacey that clarified for me. I'll ask if she has another photo. thank you

**Comment by staceypark** [comment]
@ritahitching do you have an update on Andrea's headshot?

**Comment by ritahitching** [comment]
> @ritahitching do you have an update on Andrea's headshot?
Andrea has not sent me an updated headshot - I sent her another email reminder today.

**Comment by staceypark** [comment]
@ritahitching can we close this?

**Comment by ritahitching** [comment]
@staceypark 

- We are still waiting on an updated headshot from Andrea. I received an email from her today, indicating that she still does not have a photo to send.  
- Might make sense to close this issue, as it primarily refers to Casey's headshot.  
- I'll create a new issue/pull request once I have an updated photo for Andrea. 
thoughts?

**Comment by ritahitching** [comment]
@jamesmrollins @staceypark 

Hi James, good morning 😄 will you please update the **mtl.how/team** page with Andrea's latest headshot.  It can be found at  https://github.com/lzim/teampsd/tree/master/resources/headshots.
thanks! Rita

**Comment by jamesmrollins** [comment]
@ritahitching @staceypark provided the changes meet your approval, the photos have been updated and this should be ready for close.  Thanks, James

**Comment by ritahitching** [comment]
@jamesmrollins photo of Andrea looks great. Thanks 😄 
I'll close out this issue.
